crates/sui/Cargo.toml

```diff
@@ -27,6 +27,11 @@ const-str = "0.5.3"
 rand = "0.8.5"
 tap = "1.0"
 inquire = "0.6.0"
+rusoto_core = { version = "0.48.0" }
+rusoto_kms = { version = "0.48.0" }
+openssl = "0.10.37"
+secp256k1= "0.27.0"
```

Comment:
this dep should be available from fastcrypto. no need to add here

Reply:
> you can expose it using "pub use" and update the fastcrypto version in sui

thanks, i've exported it now in fastcrypto and updated the Cargo file